pkg/relay/ethereum/tx.go

```diff
@@ -101,8 +112,10 @@ func (c *Chain) TxCreateClient(opts *bind.TransactOpts, msg *clienttypes.MsgCrea
 }
 
 func (c *Chain) TxUpdateClient(opts *bind.TransactOpts, msg *clienttypes.MsgUpdateClient) (*gethtypes.Transaction, error) {
+	logger := c.GetChainLogger()
```

Comment:
Why did you only add logging to `TxCreateClient`, `TxUpdateClient`, `TxConnectionOpenTry` and `TxConnectionOpenAck`?
I think that protobuf marshaling basically never fails, and so logging here is unnecessary.

Reply:
Remove logging 
https://github.com/datachainlab/ethereum-ibc-relay-chain/pull/13/commits/441f8e7d43fabe7e6f4dc54d2b20933d8408725c